plugins/reports/articles/ArticleReportDAO.inc.php

```diff
@@ -31,7 +31,8 @@ function getArticleReport($journalId) {
 				COALESCE(asl2.setting_value, aspl2.setting_value) AS abstract,
 				COALESCE(sl.setting_value, spl.setting_value) AS section_title,
 				a.status AS status,
-				a.language AS language
+				a.language AS language,
+				aspl1.submission_revision AS version
```

Comment:
hmmm... this seems strange to me -- are you are looking only for the revision/version of that setting_name (title?) in the primary language?
if so, what if there is no such title? -- title doesn't have to exist in the primary locale, I believe...